packages/libs/client/package.json

```diff
@@ -37,6 +37,7 @@
     "@kadena/types": "workspace:*",
     "cross-fetch": "~3.1.5",
     "debug": "~4.3.4",
+    "kadena.js": "workspace:*",
```

Comment:
Would it be hard to refactor out the dependency to kadena.js?
I'd rather depend on the specific client libs instead of generic ones like `kadena.js` and `types`

Reply:
removed kadena.js from dependency and added `create..Request` to chainweb-node-client